front/app/containers/Admin/pagesAndMenu/routes.tsx

```diff
@@ -68,11 +76,32 @@ export default () => ({
     },
     {
       path: HOMEPAGE_PATH,
-      element: <EditHomepage />,
+      element: (
+        <PageLoading>
+          <EditHomepage />
+        </PageLoading>
+      ),
     },
     {
-      path: `${CUSTOM_PAGES_PATH}/new`,
-      element: <NewCustomPage />,
+      path: CUSTOM_PAGES_PATH,
```

Comment:
Have a look at the routes and components here and see if you get why I did it this way. :-)

Reply:
Isn't it the default way of adding an index page with children?

Reply:
It depends on how we build our page. With a tab system, it is more complicated.